### ISSUE TITLE
Hyperparameters for NeRF training

### ISSUE THREAD

**Issue by yashbhalgat** [NVlabs/instant-ngp#118]
Hi @Tom94, thank you for this wonderful piece of work -- I am sure this is useful to many researchers (like me)! I have implemented the hash encoding in PyTorch and the convergence speed is quite amazing (although not as fast as your C++/CUDA impl). I was able to train a 'HashNeRF' model for **hotdog**, **chair** and few other blender objects and the model converges 50x faster than NeRF in my implementation.

Although, I am not able to get as good results for the **ficus** and **drums** objects. I see some cloudy artifacts in the final rendering. Could you please share the hyperparameters that you used for the NeRF training of these objects?

### My hyperparameters
I am using an Adam optimizer with:
- learning rate: 0.01 (tried 0.001, 0.0005)
- learning rate schedule: `lr ^ (global_step / 10000)`
- base resolution: 16
- max resolution: I tried 512, 1024, 4096
- weight decay for MLP: 1e-6 _(no weight decay for embeddings)_
- eps for embeddings: 1e-15

**Comment by Tom94** [comment]
Hi there, apologies for the delay in responding.

The hyperparameters you're listing seem to match ours. I suspect that our raymarching strategy of skipping empty space helps us avoid floaters more easily (by importance sampling the loss near surfaces). Other than that, the two implementations should behave the same.

It would probably be good for us to add an option to disable empty-space skipping to verify this. Much of the needed code is already there -- it just needs to be wired up appropriately.
